### DISCUSSION TITLE
baseline corrected tau

### DISCUSSION THREAD

**Discussion by neginhs** [jazznbass/scan#144] [Q&A] [answered]
I'm analysing the data for a SCED with multiple baselines across four participants (P1-P4). The corrected-tau results show that baseline should be corrected for P4.

1. I assumed that Tau (A vs B - trend A) has the same value as the corrected-tau, but it doesn't. Am I missing somethin here?

2. In addition to the 'scan' package, I've used the Vannest et al online tau calculator. In this calculator, you can combine the trend-corrected-tau for P4 with normal tau for P1-P3. It seems that this is not possible in 'scan'. However, I wonder if there is any way to get an overall tau across all four participants in my study using the 'scan' package. Any advise on this? 

Thanks

**Comment by jazznbass** [comment, marked as answer]
Tau-U is a quite different approach to Baselinecorrected Tau (BCT) and gives different values.


BCT is a prewhitening approach where the values of the B Phase are corrected based on a non parametric autoregression of the A phase and the extrapolated values from that on the B phase.

**Reply by neginhs** [reply]
Thanks so much for the response. May I please ask how Tau-U _A vs B - trend A_ is calculated?

**Reply by jazznbass** [reply]
There are various approaches. One approach is: 
A Kendall autocorrelation is calculated for phase A and a U-test for A vs. B is performed.
The resulting test value of the first analysis is subtracted from the test value of the second. The Tau-U value is obtained by dividing the corrected test value by the number of comparison pairs in the second analysis. 
See 
Parker, R. I., Vannest, K. J., Davis, J. L., & Sauber, S. B. (2011). Combining Nonoverlap and Trend for Single-Case Research: Tau-U. Behavior Therapy, 42(2), 284–299. https://doi.org/10.1016/j.beth.2010.08.006
for details and examples.

**Reply by neginhs** [reply]
Thanks so much for the explanation.
